src/test/java/com/softserveinc/ita/FacultiesTest.java

```diff
@@ -43,6 +43,18 @@ public void verifySearchFieldDoesntWorkWithInvalidInput() {
                 .isEqualTo(0);
     }
 
+    @Test
+    public void verifyAddingFacultyFormIsDisplayed() {
+        var actualResult
+                = facultiesPage
+                .clickOnAddFacultyButton()
+                .isAddFacultyFormIsDisplayed();
+
+        assertThat(actualResult)
+                .as("Clicking on the button 'Додати факльтет' make add faculty form appear")
```

Comment:
the weird message, in message better to use "should", "should not"

possible message: "Add faculty form should appear"